cluster/examples/kubernetes/ceph/create-external-cluster-resources.py

```diff
@@ -176,6 +176,8 @@ def gen_arg_parser(cls, args_to_parse=None):
                                   help="Provides the name of the RBD datapool")
         output_group.add_argument("--rgw-endpoint", default="", required=False,
```

Comment:
Please include similar changes for cluster/examples/kubernetes/ceph/create-external-cluster-resources.sh, May be create secret in "import-external-cluster.sh"

Reply:
Those scripts don't support the addition of rgw endpoint so I'll skip this for now.